modules/serverless-deprecated-removed-features.adoc

```diff
@@ -11,9 +11,9 @@ Some features that were Generally Available (GA) or a Technology Preview (TP) in
 For the most recent list of major functionality deprecated and removed within {ServerlessProductName}, refer to the following table:
 
 .Deprecated and removed features tracker
-[cols="3,1,1,1,1,1,1",options="header"]
+[cols="3,1,1,1,1,1,1,1",options="header"]
```

Comment:
Verify with Naina that we want to include the 1.32 column without removing any previous columns. The table no longer fits on one screen horizontally.